src/components/profile/ProfileDataTable.js

```diff
@@ -43,8 +44,14 @@ const ProfileDataTable = ({
     }
   }, [verifyEdit, profile.email, storedProfile.email])
 
-  const verifyPhone = useCallback(() => {
-    if (storedProfile.mobile && profile.mobile !== storedProfile.mobile) {
+  const verifyPhone = useCallback(async () => {
+    if (!storedProfile.mobile) {
+      const onlyCheckAlreadyVerified = true
```

Comment:
it could be hard-coded or configured via env var on the server side
i do not think we need to pass additional flags in the API call

Reply:
this is only the make the code more descriptive, so instead of calling:
`const res = await API.sendOTP({ user: profile }, true)`
the call is:
`const res = await API.sendOTP({ user: profile }, onlyCheckAlreadyVerified)`
which is easier to understand what the code is doing
I have opened a PR for GoodServer https://github.com/GoodDollar/GoodServer/pull/324 which implements this extra param in order of checking if the number is verified without sending an SMS code